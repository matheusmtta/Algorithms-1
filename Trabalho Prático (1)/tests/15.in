44 i
32 i
22 i
16 i
22 r
46 i
32 r
3 i
3 r
42 i
38 i
102 p
44 r
114 p
43 i
16 r
4 i
86 p
43 r
38 r
222 p
27 i
4 r
39 i
39 r
107 p
138 p
42 r
173 p
211 p
9 i
15 i
40 i
15 r
92 p
9 r
46 r
24 i
51 p
62 p
160 p
72 p
40 r
54 p
99 p
30 i
183 p
42 p
8 i
60 p
27 r
24 r
29 i
31 p
35 i
35 r
1 i
31 p
8 r
30 r
145 p
30 p
4 p
2 i
1 r
23 p
25 p
52 p
25 i
25 r
5 i
5 r
18 i
34 i
20 i
29 r
33 i
