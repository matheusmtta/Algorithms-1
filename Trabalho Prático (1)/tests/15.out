3
3
2
5
4
3
5
5
2
2
5
4
3
2
4
7
4
2
3
2
5
2
4
4
3
5
